src/Common/src/CoreLib/System/Diagnostics/Tracing/TraceLogging/SimpleTypeInfos.cs

```diff
@@ -159,8 +159,10 @@ public override void WriteData(TraceLoggingDataCollector collector, PropertyValu
         {
             collector.AddNullTerminatedString((string?)value.ReferenceValue);
         }
-        
+
+#pragma warning disable CS8609 // TODO-NULLABLE: this is already fixed in the latest compiler
```

Comment:
Is it fixed in 3.2.0-beta1-19229-02 (a build from two days ago)?

Reply:
Asking because the arcade toolset that brought that build in was only merged about 7 hours ago: https://github.com/dotnet/corefx/pull/37328

Reply:
@tannergooding I haven't seen this error on coreclr so I presume it's using some older version in corefx

Reply:
Right, and we just updated the build we are using in CoreFX a few hours ago, so it should be inline with what CoreCLR is using.

Reply:
will revert/rebase then and check

Reply:
ok, running in parallel locally and also pushed here

Reply:
@tannergooding nope, still needed